README.md

```diff
@@ -19,3 +9,57 @@ Make sure that you have the latest versions of Vue, Node, Vue Class Component, a
 ### How to run this application?
 
 To run this application, you need to clone the `getting-started-with-the-vue-tree-grid` repository and then open it in Visual Studio Code. Now, simply install all the necessary react packages into your current project using the `npm install` command and run your project using the `npm run serve` command.
+
+## Features and Benefits
+
+### Adaptive UI layout
+
+The [Vue Tree Grid](https://www.syncfusion.com/vue-components/vue-tree-grid?utm_source=github&utm_medium=listing&utm_campaign=vue-tree-grid-github-samples) user interface is customized and redesigned for great views and usability on small screens. Filter, search, and edit dialogs are adaptive to the screen size.
```

Comment:
Need to interlink features not only FT page.

Reply:
Done